code/index.html

```diff
@@ -1,22 +1,179 @@
+            29 0 0 0 .46-5.25 29 29 0 0 0-.46-5.33z"></path>
+            <polygon points="9.75 15.02 15.5 11.75 9.75 8.48 9.75 15.02"></polygon>
+          </svg>
+        </a>
+      </div>
     </header>
 
-    <section class="big-news">
-      <!-- The big news content -->
-    </section>
+    <div class="background-image">
+      <main>
+        <section class="big-news">
+          <section class="container">
+            <a href="#">
+              <img class="responsive-image" src="alva-wet.jpg" alt="Alva getting wet"> 
+              <div class="big-news-text">
+                <h2>Looking back</h2>
+                <p>'2020 - Year of the dog'</p>
+              </div>
+            </a>
+          </section>
+        </section>
+
+        <section class="other-news">
+          <section class="container">
+            <a href="#">
+              <img class="responsive-image" src="./alva-dot.jpg" alt="Alva in the woods"> 
+              <h3>'A casual stroll in the woods'</h3>
+            </a>
+          </section>
+          <section class="container">
+            <a href="#">
+              <img class="responsive-image" src="./alva-field.jpg" alt="Alva on a field"> 
+              <h3>'Endless possibilities - a philosophical story'</h3>
+            </a>
+          </section>
+          <section class="container">
+            <a href="#">  
+              <img class="responsive-image" src="./alva-flamingo.jpg" alt="Alva with Flamingo toy"> 
+              <h3>'How to destroy a toy in 10 seconds'</h3>
+            </a>
+          </section>
+          <section class="container">
+            <a href="#">  
+              <img class="responsive-image" src="./alva-grass.jpg" alt="Alva lying in the grass"> 
+              <h3>'Nothing beats the feeling of grass'</h3>
+            </a>
+          </section>
+          <section class="container">
+            <a href="#">  
+              <img class="responsive-image" src="./alva-tongue-out.jpg" alt="Alva with her tongue out"> 
+              <h3>'The road to happiness'</h3>
+            </a>
+          </section>
+          <section class="container">
+            <a href="#">  
+              <img class="responsive-image" src="./alva-woods.jpg" alt="Alva in the woods"> 
+              <h3>'That time I almost choked on a stick'</h3>
+            </a>
+          </section>
+          <section class="container">
+            <a href="#">  
+              <img class="responsive-image" src="./alva-snowball.jpg" alt="Alva on a big snowball"> 
+              <h3>'How I became the Queen of everything'</h3>
+            </a>
+          </section>
+          <section class="container">
+            <a href="#">  
+              <img class="responsive-image" src="./alva-sofa.jpg" alt="Alva on the sofa"> 
+              <h3>'How to relax 101'</h3>
+            </a>
+          </section>
+        </section>
+      </main>
+    </div>
+
+    <footer>
```

Comment:
Nice with a footer! Maybe you can remove the nested div and put the styling directly on the footer?